src/components/pages/index.js

```diff
@@ -22,6 +22,12 @@ function Index({ initialState }) {
   const currentHomeLanguage = homeLanguages[currentHomeLanguageIndex];
   const [isSearching, setIsSearching] = useState(false);
 
+  useEffect(() => {
+    return () => {
+      setIsSearching(false);
```

Comment:
This works on resetting the page to the false state of searching. I could also reset the input text but I think it's better if I leave it. What do you guys think?

Reply:
Not sure I get why this is needed, since after a search you navigate away.
Could you describe the scenario maybe to clarify?